### DISCUSSION TITLE
How to signup/register a new user?

### DISCUSSION THREAD

**Discussion by Jyun111** [wpengine/faustjs#814] [Q&A] [answered]
I didn't see anything related to register a new user/login on the Documentation. Is there anyway to signup/register a user?

**Comment by blakewilson** [comment, marked as answer]
Hi @Jyun111,

You are correct. Currently there isn't a mechansim to register from the frontend like `useLogin`, `useLogout`. We have a ticket in our backlog to create a`useRegister` hook.

For now, I would recommend using the `registerUser` WP GraphQL mutuation to achieve this.
